tests/empty-file/expected_results.json

```diff
@@ -1,5 +1,5 @@
 {
   "version": 1,
   "status": "fail",
-  "message": "TODO: replace with correct output"
+  "message": ""
```

Comment:
Can you provide the actual error message here?

Reply:
Building and running tests as separate steps in `run.sh` should make this possible.

Reply:
Fixed!